stetho-timber/src/main/java/com/facebook/stetho/timber/StethoTree.java

```diff
@@ -17,17 +17,38 @@
 
 /**
  * Timber tree implementation which forwards logs to the Chrome Dev console.
+ * This uses a {@link Timber.DebugTree} to automatically infer the tag from the calling class.
  * Plant it using {@link Timber#plant(Timber.Tree)}
  * <pre>
  *   {@code
  *   Timber.plant(new StethoTree())
+ *   //or
+ *   Timber.plant(new StethoTree(
+ *       new StethoTree.Configuration.Builder()
+ *           .showTags(true)
+ *           .minimumPriority(Log.WARN)
+ *           .build()));
  *   }
  * </pre>
  */
-public class StethoTree extends Timber.Tree {
+public class StethoTree extends Timber.DebugTree {
```

Comment:
Hmm, this looks to have pretty significant performance overhead, even if just for debug builds.  Also, changing the base class for `StethoTree` without any configuration option seems a bit heavy-handed since again it seems to have performance impact.

Prefer maybe a `StethoDebugTree` alternative which extends `Timber.DebugTree` and provides this functionality.